src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -104,7 +104,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
     borderRadius: 8,
-    backgroundColor: COLORS.anon_primary,
+    backgroundColor: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.anon_primary` to `COLORS.signed_primary` for the background color of the button seems fine, as long as `COLORS.signed_primary` is defined and it fits with the new color scheme you are implementing.

```diff
-    backgroundColor: COLORS.anon_primary,
+    backgroundColor: COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->